packages/tocco-ui/src/EditableValue/EditableValue.stories.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable no-console */
+import React from 'react'
+import {storiesOf} from '@storybook/react'
+import {withKnobs, number, text, boolean} from '@storybook/addon-knobs'
+import {IntlProvider} from 'react-intl'
+
+import EditableValue from './'
+import {htmlMarkup} from '../util/htmlMarkup'
+import {SelectStory} from '../Select/Select.stories'
+import {UploadStory} from '../Upload/Upload.stories'
+import {Upload as Raw} from '../Upload/Upload'
+
+const EMPTY_FUNC = () => {}
+
+storiesOf('Edit Data/EditableValue', module)
+  .addDecorator(withKnobs)
+  .add(
+    'String',
+    () => <EditableValue type="string" value={text('String', 'This is a simple string.')} onChange={EMPTY_FUNC}/>
+  )
+  .add(
+    'Text',
+    () => <EditableValue
+      type="text"
+      value={text('Text', 'This is the first line.\nTo be continued...')}
+      onChange={EMPTY_FUNC}
+    />
+  )
+  .add(
+    'Url',
+    () => <EditableValue type="url" value={text('Url', 'http://www.tocco.ch')} onChange={EMPTY_FUNC}/>
+  )
+  .add(
+    'Phone',
+    () => <EditableValue type="phone" value={text('Phone', '+41761234567')} onChange={EMPTY_FUNC}/>
+  )
+  .add(
+    'Number',
+    () =>
+      <EditableValue
+        type="number"
+        value={number('Number', 543)}
+        options={{
+          prePointDigits: 8,
+          postPointDigits: 2
+        }}
+        onChange={EMPTY_FUNC}
+      />
+  )
+  .add(
+    'Decimal',
+    () => <EditableValue
+      type="decimal"
+      value={number('Decimal', 123456.78)}
+      options={{
+        prePointDigits: 8,
+        postPointDigits: 2
+      }}
+      onChange={EMPTY_FUNC}
+    />
+  )
+  .add(
+    'Integer',
+    () => <EditableValue
+      type="integer"
+      value={number('Integer', 200)}
+      options={{
+        minValue: 0,
+        maxValue: 300
+      }}
+      onChange={EMPTY_FUNC}
+    />
+  )
+  .add(
+    'Money Amount',
+    () => <EditableValue
+      type="moneyamount"
+      value={number('Money Amount', 1234.56)}
+      options={{
+        prePointDigits: 8,
+        postPointDigits: 2
+      }}
+      onChange={EMPTY_FUNC}
+    />
+  )
+  .add(
+    'Single-Select',
+    () => <SelectStory
+      isMulti={false}
+      delay={0}
+    />
+  )
+  .add(
+    'Multi-Select',
```

Comment:
"info" box of this story is useless since its shows the stories code and proptypes.